lib/sorbet-rails/model_plugins/active_record_assoc.rb

```diff
@@ -188,6 +188,8 @@ def relation_should_be_untyped?(reflection)
 
   sig { params(reflection: T.untyped).returns(T.nilable(T::Boolean)) }
   def polymorphic_assoc?(reflection)
+    return if reflection.nil
```

Comment:
🤦‍♂️

Got ahead of myself, fixed here: https://github.com/TandaHQ/sorbet-rails/commit/68b47031cb2fac701ffcaeaa0c427be336f951fc